tests/filecheck/dialects/fsm/fsm_op.mlir

```diff
@@ -143,33 +143,4 @@
       }) {nextState = @C} : () -> ()
     }) {sym_name = "C"} : () -> ()
   }) {function_type = (i1) -> i1, initialState = "A", sym_name = "foo"} : () -> ()
-}) : () -> ()
-
-
-// CHECK:  "fsm.machine"() ({
-// CHECK: Can not find initial state
-// CHECK:  ^bb0(%arg0: i1):
-// CHECK:   %0 = "fsm.variable"() {initValue = 0 : i16, name = "cnt"} : () -> i16
-// CHECK:    "fsm.state"() ({
-// CHECK:      "fsm.output"(%arg0) : (i1) -> ()
-// CHECK:    }, {
-// CHECK:      "fsm.transition"() ({
-// CHECK:      }, {
-// CHECK:      }) {nextState = @B} : () -> ()
-// CHECK:    }) {sym_name = "B"} : () -> ()
-// CHECK:  }) {function_type = (i1) -> i1, initialState = "A", sym_name = "foo"} : () -> ()
-
-"builtin.module"() ({
-  "fsm.machine"() ({
-  ^bb0(%arg0: i1):
-    %0 = "fsm.variable"() {initValue = 0 : i16, name = "cnt"} : () -> i16
-    "fsm.state"() ({
-      "fsm.output"(%arg0) : (i1) -> ()
-    }, {
-      "fsm.transition"() ({
-      }, {
-      }) {nextState = @B} : () -> ()
-    }) {sym_name = "B"} : () -> ()
-  }) {function_type = (i1) -> i1, initialState = "A", sym_name = "foo"} : () -> ()
-}) : () -> ()
-
+}) : () -> ()
```

Comment:
```suggestion
}) : () -> ()

```
I'd recommend ticking the setting in vscode that adds trailing whitespace. I think you might also need to run `make precommit-install` or `make install-precommit`, I forget which one, on your machine

Reply:
You marked this comment as resolved, but I still see the (-) in the diff. I would recommend marking comments as resolved when they have been addressed, this makes it easier to track which changes are still expected.

Reply:
Maybe on my end it was just a vscode setting for a while, not precommit?

Reply:
I still feel like it would be good to add it manually until we amend the precommit hook